src/encode/pattern/mod.rs

```diff
@@ -536,6 +545,47 @@ impl<'a> From<Piece<'a>> for Chunk {
                         params: parameters,
                     }
                 }
+                #[cfg(feature = "log_kv")]
+                "K" | "key_value" => {
```

Comment:
Optional: We may want to warn the user on parse that the `log_kv` feature is not enabled if the user attempts to use `K`